src/com/esotericsoftware/kryo/util/IdentityObjectIntMap.java

```diff
@@ -34,11 +34,11 @@
 	private static final int PRIME3 = 0xb4b82e39;
 	private static final int PRIME4 = 0xced1c241;
 
-	public int size;
+	private int size;
 
-	K[] keyTable;
-	int[] valueTable;
-	int capacity, stashSize;
+	private K[] keyTable;
+	private int[] valueTable;
+	private int capacity, stashSize;

```

Comment:
size being private/public should not affect building. The others can be private.

Reply:
This is the error I see when I mark size private:

```
[INFO] --- clirr-maven-plugin:2.8:check (check-on-compile) @ kryo ---
[INFO] artifact com.esotericsoftware:kryo: checking for updates from sonatype-releases
[INFO] artifact com.esotericsoftware:kryo: checking for updates from sonatype-nexus-snapshots
[INFO] artifact com.esotericsoftware:kryo: checking for updates from central
[INFO] Comparing to version: 4.0.1
[ERROR] 6010: com.esotericsoftware.kryo.util.IdentityObjectIntMap: Accessibility of field size has been weakened from public to private
```